source/_integrations/refoss.markdown

```diff
@@ -2,14 +2,16 @@
 title: Refoss
 description: Integrate Refoss devices
 ha_category:
+  - Sensor
   - Switch
-ha_release: 2024.1
+ha_release: 2024.6
```

Comment:
```suggestion
ha_release: 2024.1
```

This value should not updated. It points to the version, where the integration was added with the first platform